openapi/workflow_execution_service.swagger.yaml

```diff
@@ -41,20 +41,20 @@ paths:
             $ref: '#/definitions/ErrorResponse'
       tags:
         - WorkflowExecutionService
-  /workflows:
+  /workflow-jobs:
     get:
```

Comment:
It could simply be just `/jobs` instead of `/workflow-jobs`. I personally don't see any problem either way.